eslint/base/index.js

```diff
@@ -9,9 +9,12 @@ module.exports = {
       legacyDecorators: true,
     },
   },
-  plugins: ['prettier'],
```

Comment:
`plugin:prettier/recommended` already adds `prettier` to the list of `plugins`, so this was redundant.